repo/packages/S/spark/13/marathon.json.mustache

```diff
@@ -0,0 +1,117 @@
+        "SPARK_DISPATCHER_MESOS_ROLE": "{{service.role}}",
+        "SPARK_DISPATCHER_MESOS_PRINCIPAL": "{{service.principal}}",
+        "SPARK_DISPATCHER_MESOS_SECRET": "{{service.secret}}",
+{{#security.kerberos.krb5conf}}
+        "SPARK_MESOS_KRB5_CONF_BASE64": "{{security.kerberos.krb5conf}}",
+{{/security.kerberos.krb5conf}}
+{{#security.ssl.enabled}}
+        "SPARK_SSL_ENABLED": "{{security.ssl.enabled}}",
+        "SPARK_SSL_KEYSTOREBASE64": "{{security.ssl.keyStoreBase64}}",
+        "SPARK_SSL_TRUSTSTOREBASE64": "{{security.ssl.trustStoreBase64}}",
+        "SPARK_SSL_KEYSTOREPASSWORD": "{{security.ssl.keyStorePassword}}",
+        "SPARK_SSL_KEYPASSWORD": "{{security.ssl.keyPassword}}",
+        "SPARK_SSL_TRUSTSTOREPASSWORD": "{{security.ssl.keyStorePassword}}",
+        "SPARK_SSL_PROTOCOL": "{{security.ssl.protocol}}",
+        "SPARK_SSL_ENABLEDALGORITHMS": "{{security.ssl.enabledAlgorithms}}",
+{{/security.ssl.enabled}}
+        "ENABLE_HISTORY_SERVER": "{{history-server.enabled}}",
+        "HISTORY_LOG_DIR": "{{history-server.log-dir}}",
+        "HISTORY_CLEANER_ENABLED": "{{history-server.cleaner-enabled}}",
+        "HISTORY_CLEANER_INTERVAL": "{{history-server.cleaner-interval}}",
+        "HISTORY_CLEANER_MAXAGE": "{{history-server.cleaner-maxage}}",
+        "HISTORY_KRB_PRINCIPAL": "{{history-server.kerberos.principal}}",
+        "HISTORY_KRB_KEYTAB": "{{history-server.kerberos.keytab}}",
+{{#security.mesos.authentication.enabled}}
+        "DCOS_SERVICE_ACCOUNT_CREDENTIAL": "file:///run/dcos/etc/mesos/agent_service_account.json",
+        "MESOS_MODULES": "file:///opt/mesosphere/etc/mesos-scheduler-modules/dcos_authenticatee_module.json",
+        "MESOS_AUTHENTICATEE": "com_mesosphere_dcos_ClassicRPCAuthenticatee",
+{{/security.mesos.authentication.enabled}}
+{{#security.mesos.ssl.enabled}}
+        "SSL_CERT_FILE": "/run/dcos/pki/tls/certs/mesos-slave.crt",
+        "SSL_KEY_FILE": "/run/dcos/pki/tls/private/mesos-slave.key",
+        "SSL_CA_DIR": "/run/dcos/pki/certs",
+{{/security.mesos.ssl.enabled}}
+        "SSL_ENABLED": "{{security.mesos.ssl.enabled}}"
+    },
+    "ports": [
+        0,
+        0,
+        0,
+        0
+    ],
+    "container": {
+        "type": "DOCKER",
+        "docker": {
+            "image": "{{resource.assets.container.docker.spark_docker}}",
+            "network": "HOST",
+            "forcePullImage": true
+        },
+        "volumes": [
+            {
+                "hostPath": "/run/dcos/pki/",
+                "containerPath": "/run/dcos/pki/",
+                "mode": "RO"
+            }
+{{#security.mesos.authentication.enabled}}
+            ,
+            {
+                "hostPath": "/opt/mesosphere/",
+                "containerPath": "/opt/mesosphere/",
+                "mode": "RO"
+            },
+            {
+                "hostPath": "/run/dcos/etc/mesos/",
+                "containerPath": "/run/dcos/etc/mesos/",
```

Comment:
Do you still need this or /opt/mesosphere if authentication is disabled but mesos.ssl is enabled?